docs/guides/hello-world.mdx

```diff
@@ -38,25 +38,13 @@ To get started with [Configu Platform](https://app.configu.com/), you'll need to
 ### 4. Upsert Configs to ConfigSet
 
 ```shell
-configu upsert \
-  --store 'configu' --set 'development' --schema './start.cfgu.json' \
-  --config "GREETING=hey" --config "SUBJECT=<YOUR_NAME>" \
-&& configu upsert \
-  --store 'configu' --set 'development/qa' --schema './start.cfgu.json' \
-  -c "GREETING=hola"
-&& configu upsert \
-  --store 'configu' --set 'production' --schema './start.cfgu.json' \
-  -c "GREETING=welcome"
+configu upsert --store 'configu' --set 'development' --schema './start.cfgu.json' --config "GREETING=hey" --config "SUBJECT=<YOUR_NAME>" && configu upsert --store 'configu' --set 'development/qa' --schema './start.cfgu.json' -c "GREETING=hola" && configu upsert --store 'configu' --set 'production' --schema './start.cfgu.json' -c "GREETING=welcome"
```

Comment:
Instead of one-line-ing everything, I think it's more readable to correct the command by adding the missing `\` at the end of line 46 in the original code.

I always find that long commands like the `configu upsert` ones tend to be easier to read across multiple lines rather than a single line. Particularly in IDEs :sweat_smile:

Reply:
@Aman1919 @jamesL92 is absolutely right—it will be much easier to read as multiline. Thanks for renaming the schema flag! Please keep the multiline string format and fix the missing slash at line 46.

Reply:
@rannn505 @jamesL92 thank for your review . i have done the changes